soaks/bin/build_container.sh

```diff
@@ -3,10 +3,11 @@
 set -o errexit
 set -o pipefail
 set -o nounset
-#set -o xtrace
+set -o xtrace
 
 __dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-SOAK_ROOT="${__dir}/.."
+ROOT="${__dir}/../../"
+SOAK_ROOT="${ROOT}/soaks"
```

Comment:
Is this a different directory than before?

Reply:
It comes to the same thing, I believe, but I can compute `SOAK_ROOT` in terms of `__dir` and leave that alone.

Reply:
Resolved in 464f1ef